### DISCUSSION TITLE
Documentation sharing : sqlite schema.

### DISCUSSION THREAD

**Discussion by meteos77** [textbrowser/biblioteq#290] [Show and tell]
![2023-04-28_BiblioteQ_Schema-table-book](https://user-images.githubusercontent.com/56431039/235194708-a0ae18b3-eb84-4037-8b08-fe8d7d38b325.png)

**Comment by meteos77** [comment]
![2023-04-28_BiblioteQ_Schema-table-member](https://user-images.githubusercontent.com/56431039/235193483-0065102d-45f6-45cd-995d-54951d3b01da.png)

**Comment by textbrowser** [comment]
This is very nice. Was this automatically generated from the description?

**Reply by meteos77** [reply]
https://dbschema.com/ - automatically

**Reply by textbrowser** [reply]
> https://dbschema.com/ - automatically

Very nice. I'll add the images to BiblioteQ.odt and for other projects.

**Comment by textbrowser** [comment]
One of the reminders is to document the relationships. I created such a diagram many years ago but it was painful.

**Comment by meteos77** [comment]
![2023-04-28_BiblioteQ_Schema-cd](https://user-images.githubusercontent.com/56431039/235201697-5828ff83-9868-418d-a1f6-7a5558208a5c.png)

**Comment by meteos77** [comment]
![2023-04-28_BiblioteQ-Schema-cd_formats](https://user-images.githubusercontent.com/56431039/235201735-7e1ce61c-6683-4d61-98fd-74eb304fd704.png)

**Comment by meteos77** [comment]
![2023-04-28_BiblioteQ_Schema-dvd](https://user-images.githubusercontent.com/56431039/235201762-33dcf88d-7f6f-4f9d-8ba3-6dbff384837b.png)

**Comment by meteos77** [comment]
![2023-04-28_BiblioteQ_Schema-grey_literature](https://user-images.githubusercontent.com/56431039/235201823-235322a6-2dba-4df9-9d9f-fac1a9a4f0b9.png)

**Comment by meteos77** [comment]
![2023-04-28_BiblioteQ_Schema-journal](https://user-images.githubusercontent.com/56431039/235201854-0a9add4a-7b74-4a01-8725-5884c535e9fa.png)

**Comment by meteos77** [comment]
![2023-04-28_BiblioteQ_Schema-magazine](https://user-images.githubusercontent.com/56431039/235201888-ec40961b-a8fb-46ff-b6dc-caf234fa375d.png)

**Comment by meteos77** [comment]
![2023-04-28_BiblioteQ_Schema_photograph](https://user-images.githubusercontent.com/56431039/235201924-8c99f8ef-c69b-4cba-a174-4fec9f53bb2d.png)

**Comment by textbrowser** [comment]
This is very similar to what I drew many years ago but not so fancy and certainly not mandatory. This is generated so it's always current.

**Comment by textbrowser** [comment]
Very nice discovery.

**Comment by meteos77** [comment]
![2028-04-28_BiblioteQ_Schema-videogame](https://user-images.githubusercontent.com/56431039/235202849-9b079aef-9390-4ae3-98cc-c251595052d4.png)

**Comment by meteos77** [comment]
the 10 in .png format
[2023-04-28_pictures_schema-sqlite.zip](https://github.com/textbrowser/biblioteq/files/11355541/2023-04-28_pictures_schema-
sqlite.zip)

**Comment by textbrowser** [comment]
This is a good representation of the complexity and abundance of BQ databases.

**Comment by textbrowser** [comment]
Brew has this! (Mac)

**Comment by textbrowser** [comment]
![BQ@PQ](https://user-images.githubusercontent.com/10701156/235205397-3847f5ff-87c0-4587-8a07-bc4361068c26.png)

**Reply by meteos77** [reply]
The difference is that you work on all the tables :-)

**Comment by textbrowser** [comment]
I wonder if there are other cataloging projects which support two database engines.

**Reply by meteos77** [reply]
I only studied 3 programs and only BQ had SQL for functionality.

**Comment by textbrowser** [comment]
Just three?

**Reply by meteos77** [reply]
I eliminated those that were client-server database at the beginning.

**Comment by textbrowser** [comment]
True, there are very few open source cataloging projects. There is Evergreen and Koha and some others. But those require networks and are generally huge to manage. BQ doesn't. You should create a PQ to SQ tool. :D

**Reply by meteos77** [reply]
What is SQ?

**Reply by textbrowser** [reply]
SQLite.

**Comment by textbrowser** [comment]
BQ is easy to establish and operate.

**Reply by meteos77** [reply]
In my comparison table, out of 53 points 

BQ has 42,5 and its "competitor" 30 :-)

**Reply by meteos77** [reply]
you make it more and more library management

**Comment by textbrowser** [comment]
Only 81%? Europeans measure quality through tables. BQ has unknown features because they're not documented. Do you have an Easter Egg metric?

**Reply by meteos77** [reply]
81% is still great -  today 2% with BA/Return :-)

BQ is improving and some features are not present compared to (either our old software which was our work reference.) or to the functionality that I could test on other software that would be great to have in BQ.

**Reply by meteos77** [reply]
Easter egg in chocolate is 100%.

**Reply by meteos77** [reply]
I will translate my spreadsheet to show the features.

**Comment by textbrowser** [comment]
You should include free support and almost-real-time responses. When I die, no one will support BQ and that's that. Perhaps people will love it and adopt it or not.

**Reply by meteos77** [reply]
The 81% is on the features present.

Satisfaction with the programmer is inescapable.

**Reply by textbrowser** [reply]
The future will be all stupid text programs replying to tickets.

**Reply by textbrowser** [reply]
Like when you need mobile help from Verizon. Wait two hours, try this, and then wait another 10 days.

**Reply by textbrowser** [reply]
Or when you buy a video game, the company creates a shipping label, and never ships. Zero people to contact. Cannot cancel it. So what do humans do? Don't buy online! Or contact your bank / credit company and delegate.

**Reply by meteos77** [reply]
Buy has serious stores with phone and salesmen.

In America you are always 10 years ahead of us, for better or for worse

**Reply by textbrowser** [reply]
American innovation varies by state and municipality. Some states are backwards as can be.

**Comment by textbrowser** [comment]
https://stackexchange.com/search?q=biblioteq

**Reply by meteos77** [reply]
Great! BQ advice people.

**Comment by textbrowser** [comment]
Easter Egg this.

https://en.wikipedia.org/wiki/Easter_egg_(media)

**Reply by meteos77** [reply]
Easter Egg in BQ ?

**Reply by textbrowser** [reply]
Yes, like drag-and-drop images.

**Reply by textbrowser** [reply]
Barcode generator.

**Reply by meteos77** [reply]
?

**Comment by textbrowser** [comment]
The future will be grim stupid robots with zero people. Just frustration.

**Reply by meteos77** [reply]
Let's enjoy the present time then :-)

**Comment by textbrowser** [comment]
The future must be made safe.

**Comment by textbrowser** [comment]
Less automation is good.

**Reply by meteos77** [reply]
You know, I live in the country at the end of my street there are cows,
 I like to walk in the forest, nature ... then the automation ...

**Reply by textbrowser** [reply]
Research topic: do trees decrease crime and mental illness? Probably.

**Reply by textbrowser** [reply]
If you don't have trees, libraries are mandatory.

**Reply by meteos77** [reply]
I can't live without seeing trees.

**Reply by textbrowser** [reply]
Was Arbor Day?

**Reply by textbrowser** [reply]
https://en.wikipedia.org/wiki/Arbor_Day

Not present in France.

**Reply by meteos77** [reply]
In France, nothing in some cities but not nationally.

in my village, we have a woodcutter contest :-(

**Reply by textbrowser** [reply]
To kill trees? Paul Bunyan?

**Reply by meteos77** [reply]
yes, I am in a forestry commune where the exploitation of the forest is part of the local work.

**Reply by meteos77** [reply]
surface 50% forest and 35% meadow

**Reply by textbrowser** [reply]
Western Europe kind of exploits Eastern Europe for timber. There was a Netflix documentary covering corrupt logging.

**Reply by textbrowser** [reply]
IKEA was a culprit.

**Reply by meteos77** [reply]
our wood is sent to china :( the sawmills had no more work because of the lack of supply of logs.

Tell me the area where everything is clean.

**Reply by meteos77** [reply]
Have a nice day.

**Reply by textbrowser** [reply]
You also. See you.
